docs/content/menu/index.md

```diff
@@ -6,6 +6,8 @@ headless: true
 - [Tech radar]({{< ref "/docs/tech_radar/TechRadar.md" >}})
 - [Known Issues]({{< ref "/docs/KnownIssues.md" >}})
 - [Contacts]({{< ref "/docs/Contacts.md" >}})
+- Plugins
```

Comment:
The proposal is to separate domains:
- Reusable products (standalone plugins, test runner, test library) 
- Avito specific sections (Testing, Architecture, Analytics)

Reply:
`Plugins` is too specific for proposed section
Maybe 
```
Projects
  - Build checks gradle plugin
```

Reply:
Agree, projects is better.

Reply:
Fixed.
Let's start with a simple "Build checks". We will have many plugins and it may look noisily on the menu.
Anyway, we can change it easily later. It won't change URLs.